crates/revm/Cargo.toml

```diff
@@ -19,22 +19,32 @@ hex = { version = "0.4", optional = true }
 num_enum = { version = "0.5", default-features = false } # used for SpecId from u8 cast
 revm_precompiles = { path = "../revm_precompiles", version = "1.1.2", default-features = false }
 rlp = { version = "0.5", default-features = false } # used for create2 address calculation
-ruint = { version = "1.7.0", features = ["primitive-types", "rlp",]  }
-serde = { version = "1.0", features = ["derive","rc"], optional = true }
-tokio = { version = "1.21", features = ["rt-multi-thread", "macros"], optional = true }
-web3 = { git = "https://github.com/tomusdrw/rust-web3", version = "0.19", optional = true }
+ruint = { version = "1.7.0", features = ["primitive-types", "rlp"] }
+serde = { version = "1.0", features = ["derive", "rc"], optional = true }
+tokio = { version = "1.21", features = [
+    "rt-multi-thread",
+    "macros",
+], optional = true }
+ethers = { version = "1.0.2", optional = true }
```

Comment:
If you want tk be sensitive about bundle size I guess you could make it work with just ethers-providers

Reply:
@0xDmtri it would be good to trim it.

Reply:
Yes, sure, will fix it right now, sounds good!